### PR TITLE
Wdc 16- Services Completed

### PR DESCRIPTION
**Pull request by oleend**
Finished the Services Tim! Let me know if there is anything you want to change!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,133 +1,204 @@
...
 <html lang="en">
-<head>
-    <meta charset="utf-8"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
```
Comment (wojo1086) [inline index.html:8]: Please revert this back to be on one line. Try not to change the layout of the existing code until we decide on a coding standard.

File: index.html [hunk 2]
```diff
@@ -1,133 +1,204 @@
...
     <title>Web Dev Coop</title>
     <!-- CSS only -->
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
-    <link href="styles/main.css" rel="stylesheet">
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
```
Comment (wojo1086) [inline index.html:17]: Here too.

File: index.html [hunk 3]
```diff
@@ -1,133 +1,204 @@
...
+    <link href="styles/main.css" rel="stylesheet" />
     <!-- JavaScript Bundle with Popper -->
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
-    <script src="https://kit.fontawesome.com/d847726f39.js" crossorigin="anonymous"></script>
-</head>
-<body>
-<nav class="navbar navbar-expand-lg navbar-light fixed-top">
-    <div class="container-fluid">
+    <script
```
Comment (wojo1086) [inline index.html:20]: And here.

File: index.html [hunk 4]
```diff
@@ -1,133 +1,204 @@
...
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p"
+      crossorigin="anonymous"
+    ></script>
+    <script
```
Comment (wojo1086) [inline index.html:25]: And here.

**Review by wojo1086 — CHANGES_REQUESTED** [review]
Please revert all the extra spacing changes. It's making it difficult to see what you actually changed in your code.

File: index.html (hunk 1 above)
Comment (oleend) [reply index.html:8]: This looks different then what is shown in github? Its on one line on my editor

File: index.html (hunk 2 above)
Comment (oleend) [reply index.html:17]: Guess i need that editor plugin

File: index.html (hunk 3 above)
Comment (oleend) [reply index.html:20]: I can now see what you see!

Comment (oleend) [reply index.html:20]: Ready to pull!

**Review by wojo1086 — APPROVED** [review]

**Pull request merged**